hazelcast/src/hazelcast/client/network.cpp

```diff
@@ -132,17 +132,17 @@ namespace hazelcast {
             }
 
             void ClientConnectionManagerImpl::schedule_connect_to_all_members() {
+                if (!client_.get_lifecycle_service().is_running()) {
+                    return;
+                }
+
```

Comment:
This change doesn't actually affect anything, right? I assume it's just for consistency with the other one.

Reply:
yes, it doesn't change anything.